runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -560,9 +561,18 @@ private void hop(boolean previous)
 
 			world = worlds.get(worldIdx);
 
-			// Check world region if filter is enabled
-			if (config.quickHopRegionFilter() != RegionFilterMode.NONE && world.getRegion() != config.quickHopRegionFilter().getRegion())
+			// Check if quick-hop is disabled.
+			if (!config.quickHopAURegion() && !config.quickHopDERegion()
+					&& !config.quickHopGBRegion() && !config.quickHopUSRegion())
 			{
+				return;
+			} // Check world region if filters are enabled
```

Comment:
This comment is mostly superfluous given the obviousness of the conditional below, so I don't think it's needed.
```suggestion
			}
```